pkgs/development/compilers/elm/default.nix

```diff
@@ -23,6 +23,11 @@ let
                 wrapProgram $out/bin/elm \
                   --prefix PATH ':' ${lib.makeBinPath [ nodejs ]}
               '';
+
+              description = "A delightful language for reliable webapps.";
```

Comment:
```suggestion
              description = "Delightful language for reliable webapps";
```

Reply:
This is taken from https://elm-lang.org/ - I'm not sure if it makes sense to change this.

Reply:
I've removed trailing `.` but kept `A` as it is the official tagline.